matrix/buffer.py

```diff
@@ -904,7 +904,7 @@ def __init__(self, room, server_name, homeserver, prev_batch):
 
         self.last_message = None
 
-        buffer_name = "{}.{}".format(server_name, room.room_id)
+        buffer_name = "matrix.{}.{}".format(server_name, room.room_id)
```

Comment:
Let's use `SCRIPT_NAME` for consistency sake.

Reply:
Yes, sorry, I actually realised that myself just now :p